### ISSUE TITLE
settings button showing empty values homer10-app

### ISSUE THREAD

**Issue by bilalrao12** [sipcapture/homer-docker#11]
When we click on this setting buttons:
![image](https://github.com/sipcapture/homer-docker/assets/118199337/554a19e9-8842-494d-964b-d25d60f21afa)

It shows nothing:
![image](https://github.com/sipcapture/homer-docker/assets/118199337/846212ad-976a-4e5e-be03-a2f568253e5a)

**Comment by rnalrd** [comment]
Where is found homer10-app? I can't find it on homer-docker repo. What DB backend is using?

**Comment by lmangani** [comment]
@rnalrd you're right there's no such thing - this refers to the grafana-flow plugin included in the HOMER10 bundle.
